userguide/one_shot_object_detection/README.md

```diff
@@ -27,15 +27,15 @@ test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'),
 # Create a model. This step may take several hours                                      
 model = tc.one_shot_object_detector.create(starter_images, 'label')
 
-# Save predictions on the test set as a column of the test data SFrame
+# Save predictions on the test set
 test_data['predictions'] = model.predict(test_data)
 
 # Draw prediction bounding boxes on the test images
 test_data['image_with_predictions'] = \
 	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions']) 
 
-# Evaluate the model and save the results in a dictionary
-metrics = model.evaluate(test_data)
+# To visualize the predictions made on the test set
+test_data.explore()
 
 # Save the model for later use in TuriCreate
 model.save('mymodel.model')
```

Comment:
Use another name other than my model and my detector. Something like 'MyStopSignDetector'

Reply:
I was following the model naming convention in the other toolkit user-guides. Apart from DC and Text classifier, all toolkits save the model as `mymodel.model` and the coreml model as `MyCustom***.mlmodel`. Should I care about following this convention?

Reply:
You can leave it as is.